services/hanke-service/src/main/kotlin/fi/hel/haitaton/hanke/validation/HankePublicValidator.kt

```diff
@@ -33,7 +33,7 @@ object HankePublicValidator {
             .andAllIn(hanke.alueet, "alueet", ::validateAlue)
             .and { notEmpty(hanke.omistajat, "omistajat") }
             .andAllIn(hanke.omistajat, "omistajat", ::validateYhteystieto)
-            .andAllIn(hanke.arvioijat, "arvioijat", ::validateYhteystieto)
+            .andAllIn(hanke.rakennuttajat, "rakennuttajat", ::validateYhteystieto)
```

Comment:
Should `hanke.muut` be validated here as well?

Reply:
Added this.